app.js

```diff
@@ -0,0 +1,26 @@
+require('dotenv').config();
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const cors = require('cors');
+let MongoDB = require('./database/db');
+let routes = require('./routes/routes.js')
+const PORT = process.env.PORT;
+
+MongoDB.config();
+
+app.use(cors());
+app.options('*', cors());
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
Do we need urlencoded parsing?

Reply:
Yep, we actually don't. :) If we end up we need it when we create routes to query one more than one param, we'll add it then.